client/src/index.css

```
@@ -4,68 +4,129 @@
+    --sidebar-muted: 240 5% 78%;
+  }
+  * {
+    @apply border-border;
+  }
+  html {
+    font-size: 16px;
+    text-rendering: optimizeLegibility;
+  }
+  body {
+    @apply bg-background text-foreground antialiased;
+    font-family: "IBM Plex Sans", system-ui, -apple-system, sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)